node/pkg/watchers/solana/ccq.go

```diff
@@ -37,7 +41,8 @@ func (w *SolanaWatcher) ccqHandleQuery(ctx context.Context, queryRequest *query.
 
 	switch req := queryRequest.Request.Query.(type) {
 	case *query.SolanaAccountQueryRequest:
-		w.ccqHandleSolanaAccountQueryRequest(ctx, queryRequest, req)
+		giveUpTime := start.Add(query.RetryInterval).Add(-250 * time.Millisecond)
```

Comment:
Does the RetryInterval need to be > 250mS?

Reply:
If RetryInterval < 250mS, then ccqCheckForMinSlotContext() will give up and return false and just not do a retry?  Sounds fair?

Reply:
Added a unit test to ensure this.